src/jetson/main.py

```diff
@@ -13,10 +13,10 @@
 from src.jetson.classifier import Classifier
 from src.jetson.encryptor import Encryptor
 from src.db import data_insertion
+from scripts import name_giver
```

Comment:
we should never have to import from the scripts folder. This means that code needs to be moved around.

Reply:
What is the scripts folder dedicated specifically for? I just put name_giver.py in there because it's very general purpose and not related very much to the rest of our src folder. I can move it there if that's better

Reply:
so I would define the scripts folder as a place for standalone programs that directly use anything in `src`. a great example would be a script that trains the model. its not the main program but a standalone that directly uses code from `src`.